clang/test/Driver/sycl-offload-save-temps.cpp

```diff
@@ -40,3 +40,10 @@
 // RUN: | FileCheck %s --check-prefix=CHK_LLVM_PASSES
 // CHK_LLVM_PASSES-NOT: clang{{.*}} "-triple" "spir64-unknown-unknown" {{.*}} "-disable-llvm-passes"
 // CHK_LLVM_PASSES: clang{{.*}} "-triple" "x86_64-unknown-linux-gnu" {{.*}} "-disable-llvm-passes"
+
+/// Usage of -save-temps with a file in $CWD should create the intermediate
+/// files _not_ in /tmp.
+// RUN: touch dummy.c
+// RUN: %clangxx -save-temps dummy.c -### 2>&1  \
```

Comment:
Shouldn't here be one of `-fsycl-*` options?

Reply:
In case this patch fixes non-SYCL mode, probably this case should be added to generic C/C++ test file.
This one (`clang/test/Driver/sycl-offload-save-temps.cpp`) is supposed to test SYCL offload mode. Right?

Reply:
Right - this impacts regular usage, not specific to `-fsycl`.  This was broken when making changes specific to SYCL offload.